pkg/kubelet/util.go

```diff
@@ -118,3 +119,12 @@ func allowHostIPC(pod *api.Pod) (bool, error) {
 	}
 	return false, nil
 }
+
+// dirExists returns true if the path exists and represents a directory.
+func dirExists(path string) bool {
```

Comment:
suggestion: this function seems general enough. How about moving it to `pkg/kubelet/util/` ?